CMakeLists.txt

```diff
@@ -406,6 +406,19 @@ function(auto_test target)
   endif()
 endfunction()
 
+function(auto_test_cc target)
```

Comment:
Maybe we can pass the suffix and avoid duplicating this? It's literally 1 byte difference between the two macros.

Reply:
Does this work for you?